readalongs/align.py

```diff
@@ -911,7 +911,15 @@ def save_readalong(
 
     if "html" in output_formats:
         html_out_path = output_base + ".html"
-        html_out = create_web_component_html(tokenized_xml_path, smil_path, audio_path)
+        html_out = create_web_component_html(
+            tokenized_xml_path,
+            smil_path,
+            audio_path,
+            config.get("title", "Title goes here"),
+            config.get("header", "Header goes here"),
+            config.get("subheader", "Subheader goes here"),
```

Comment:
I might put in default title/header/subheader that's going to be suitable as a title/header/subheader for most RAs, but I'm not sure what such good default might be.

Oh wait! Those were already the defaults we were generating, you just moved them out to the config! Well, usable defaults would have value, but they're certainly not required for this PR.

Reply:
More importantly, it would be good to use these in `text.save_minimal_index_html()`, by the time we accept them as parameters in the config. Is there anywhere else this information should get plugged in?